alphastats/DataSet.py

```diff
@@ -189,6 +168,12 @@ def load_metadata(self, file_path):
             df = file_path
         #  loading file needs to be more beautiful
         elif file_path.endswith(".xlsx"):
```

Comment:
How do you know this is the only time this warning is raised and supposed to be ignored?

Reply:
trial and error ;-)